tarmac/src/main/scala/nl/lumc/sasc/biopet/pipelines/tarmac/Tarmac.scala

```diff
@@ -66,17 +73,100 @@ class Tarmac(val parent: Configurable) extends QScript with PedigreeQscript with
 
     val wisecondorRefJobs = refMap map {
       case (sample, refSamples) =>
-        val refDir = new File(new File(outputDir, sample.sampleId), "wisecondor_reference")
-        createWisecondorReferenceJobs(refSamples, refDir)
+        val refDir = new File(sample.wisecondorDir, "wisecondor_reference")
+        sample -> createWisecondorReferenceJobs(refSamples, refDir)
     }
 
     val xhmmRefJobs = refMap map {
       case (sample, refSamples) =>
-        val refDir = new File(new File(outputDir, sample.sampleId), "xhmm_reference")
-        createXhmmReferenceJobs(sample, refSamples, refDir)
+        val refDir = new File(sample.xhmmDir, "xhmm_reference")
+        sample -> createXhmmReferenceJobs(sample, refSamples, refDir)
+    }
+
+    val wisecondorZJobs = wisecondorRefJobs map {
```

Comment:
Making vals for methods inside a method is not really usefull. This is usefull if you want to have access to the jobs output this class but in this way the collections of addAll's a just a bit weird